.vscode/cspell.global.yaml

```diff
@@ -95,6 +95,8 @@ ignoreWords:
     - aztfmod
     - menuid
     - PLACEHOLDERIACTOOLS
+    - PLACEHOLDERTITLE
+    - PLACEHOLDERDESCRIPTION
```

Comment:
wondering if it might be better to use snack case like `PLACEHOLDER_DESCRIPTION` instead?  so its easier to read and we wouldn't have to add the words to the dictionary.  
Would it still work for repoman?

Reply:
updated